tests/local_mode/test_store.py

```diff
@@ -63,3 +63,13 @@ def test_get_string_for_key():
 
     # Check if both strings are same
     assert string == returned_string
+
+
+def test_dynamic_store_size():
```

Comment:
Can you please put the test back, the one that shows how the string store grows in size.

After @Nilanshrajput finished his PR, we might remove this test or adjust it. But for the moment it will be useful